packages/common/src/DateUtils.test.ts

```diff
@@ -67,4 +68,19 @@ describe("DateUtils", function () {
       new Date(2022, 0, 2, 11, 0)
     );
   });
+
+  test("getNumberOfDaysBetweenDates works", () => {
+    const DAYS_DIFFERENCE = 10;
+    const now = new Date();
```

Comment:
Please use a const date.
i.e. something like `now = new Date(1649927794000)`.

This makes the tests deterministic

Reply:
Although I am using a const to test difference ? Why should it not be deterministic that way ?

Reply:
I agree that in this specific test this is not super critical, but what I mean here is a general idea in unit tests. 
Tests should behave and run **exactly the same** every time they run, no matter when they run.

Let's say javascript has some weird bug tomorrow that when `new Date()` is called at times the moon is full, it returns undefined. In this case, our unit test would fail once a month, even if we don't change the code it is testing.

I know this is a really weird example, but my point here is that since we call `new Date()` here, this test relies on the time it is run (as `new Date()` returns a different value based on the time it runs).
If you change the call to something like the first comment here, the test would be more deterministic and the run would be more similar every time it runs